counterpartylib/lib/api.py

```diff
@@ -764,6 +779,255 @@ def handle_root(args_path):
                 # Not found
                 return flask.Response(None, 404, mimetype='application/json')
 
+        @dispatcher.add_method
+        def sendrawtransaction(tx_hex):
+            micropayments.validate.tx_signature(dispatcher, tx_hex)
+            return backend.sendrawtransaction(tx_hex)
+
+        #########################
+        # Micropayment channels #
+        #########################
+
+        @dispatcher.add_method
+        def mpc_make_deposit(asset, payer_pubkey, payee_pubkey,
+                             spend_secret_hash, expire_time, quantity):
+            """ Create deposit and setup initial payer state.
+
+            Args:
+                asset (str): Counterparty asset.
+                payer_pubkey (str): Hex encoded public key in sec format.
+                payee_pubkey (str): Hex encoded public key in sec format.
+                spend_secret_hash (str): Hex encoded hash160 of spend secret.
+                expire_time (int): Channel expire time in blocks.
+                quantity (int): Asset quantity for deposit.
+
+            Returns:
+                {
+                    "state": dict,   # Initial payer micropayment state object.
+                    "topublish": str # Unsigned deposit rawtx.
+                }
+            """
+            netcode = "XTN" if config.TESTNET else "BTC"
```

Comment:
this might be better as a constant either in this file (`APIServer`?) or in a `micropayments_core.config` or `micropayments_core.util`...